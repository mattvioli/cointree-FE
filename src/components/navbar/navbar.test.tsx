import React from "react";
import { render, screen } from "@testing-library/react";
import { Navbar } from "./index";
describe("Navbar component", () => {
  it("renders the Cointree Logo image correctly", () => {
    render(<Navbar />);
    const logo = screen.getByAltText("Cointree Logo");
    expect(logo).toBeInTheDocument();
  });

  it("has the correct Tailwind classes applied", () => {
    render(<Navbar />);
    const navbar = screen.getByTestId("navbar");

    expect(navbar).toMatchSnapshot();
  });
});
