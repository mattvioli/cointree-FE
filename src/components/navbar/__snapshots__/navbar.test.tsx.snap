// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component has the correct Tailwind classes applied 1`] = `
<div
  class="flex flex-row justify-center z-10 w-full items-center font-mono text-sm lg:flex bg-white min-h-[3.25rem] shadow-lg"
  data-testid="navbar"
>
  <div
    class="w-full max-w-6xl"
  >
    <img
      alt="Cointree Logo"
      class="dark:invert"
      height="24"
      src="/cointree.svg"
      width="100"
    />
  </div>
</div>
`;
